src/components/AddressForm/AddressForm.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import { createCustomerRecord, redirectToCheckout } from '../../common/api';
+
+const initialValues = {
+  email: '',
+  first: '',
```

Comment:
We can drop first last and email - Stripe will get that - just need the address.

Although maybe there's a way to just do it here and then pre-populate Stripe's form?

Reply:
The email at least is already being sent with the Stripe redirect and pre-populated (https://github.com/kylemh/loven/pull/4/files#diff-13353fff3701307a37ec88a876753e35R18)

I just thought it seemed a bit awkward to ask for address and whatnot and not the name, that's not something you normally see.

Reply:
I agree. Is there any way to pass the name along too?